system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
well it will not work with openrc, I guess we need two files if we do not use copytruncate.

but restarting the service will damage user interface, no? do we really want to restart?